bots/tests-npm-updater

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+set -ex
+
+npm install --save git+https://github.com/${TEST_HEAD_REPO}#${TEST_HEAD_REF}
```

Comment:
Please quote the argument; while shell *should* do the right thing and not interpret the `#` as a comment, it's confusing to look at IMHO.